src/app/settings/index.component.ts

```diff
@@ -289,8 +291,17 @@ export class IndexComponent implements OnInit {
     this.#navigationService.saveSidebar(this.sidebar);
     this.keys = this.#sortKeys(this.#storageService.restoreKeys());
   }
-
+  
+  /**
+   * Remove an item from the sidebar item array with his index number.
+   * Except when equal to "settings", settings item must be available for an easier users navigation.
+   * @param {number} index Index of selected sidebar item
```

Comment:
We don't care (and TypesScript will not use them) of these JSDoc types since we use TypeScript. Using it will just give you more work it you change the function.

Reply:
I agree with you but it's a requirement from @ngruelaneo for PR validation.